packages/adapter/addon/json-api.js

```diff
@@ -156,7 +156,7 @@ const JSONAPIAdapter = RESTAdapter.extend({
     options.contentType = 'application/vnd.api+json';
 
     let hash = this._super(url, type, options);
-    hash.headers['Accept'] = 'application/vnd.api+json';
+    hash.headers['Accept'] = options.Accept || 'application/vnd.api+json';
```

Comment:
Imo what runspired proposed would solve this better, something like:

```
    options.contentType = options.contentType || 'application/vnd.api+json';

    let hash = this._super(url, type, options);
    hash.headers['Accept'] = options.contentType;
```

Reply:
We don't want to mix `accept` and `contentType`. I made a mistake in the issue when linking to `contentType` because I failed to notice the issue was for `accept`.

That said, we should fix both:

```js
options.contentType = option.contentType || 'application/vnd.api+json';

let hash = this._super(url, type, options);
hash.headers['Accept'] = hash.headers['Accept'] || 'application/vnd.api+json';
```

Reply:
Thanks @makepanic for the review. 

 I see. However, I am a little confused here - may be I am missing something, but using the value of "options.contentType" for "hash.headers['Accept'] may be confusing for the calling function.

Consider the following invocation of this method:

`adapter.ajaxOptions('example.com', 'GET', {'Accept': 'application/json'});`

In this case, only the 'Accept' header is being sent - 'contentType' is not being sent. So, the following code will set the hash.headers['Accept'] to 'application/vnd.api+json', even when the caller has specified 'application/json' as the value for the Accept header.
 
```
options.contentType = options.contentType || 'application/vnd.api+json';
let hash = this._super(url, type, options);
hash.headers['Accept'] = options.contentType;

```